compiler/luci/pass/src/PassTestUtils.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PASS_TEST_UTILS_H__
```

Comment:
This file name is somewhat general. What are your plans of the purpose of this file?
If this file is use only for testing, why not follow it.

1) as there is only one class, maybe follow that class name
2) maybe `PassTestGraphs.h`

Reply:
> This file name is somewhat general. What are your plans of the purpose of this file?

I intentionally used a general name. I'd like to make this file have all utility functions/classes related to testing luci passes. Other tests may use them by including this file.

Currently there is only one class (which will be used by constant folding passes), but more functions/classes will be added in this file.

Reply:
> I'd like to make this file have all utility functions/classes related to testing luci passes.

In the past, I've done like this and now don't do this for some reasons...  So I would say I'm against this.
Could you please discuss with @parjong with this topic and listen what he might say?

Reply:
> Could you please discuss with @parjong with this topic and listen what he might say?

I asked @parjong. His advice was not to make a file with ambiguous purposes. Such a file may become very large, thus difficult to manage. I understand his concern.

>2. maybe PassTestGraphs.h

I changed the file name to `PassTestGraphs.h` because it looks more specific. Thanks for the good naming.